.gitignore

```diff
@@ -0,0 +1,2 @@
+*.o
+metamath
```

Comment:
The reason why I added this comment is, because the metamath sources are distributed via a tar archive as well. Obviously you wont need the gitignore in this case. I think, some kind of how-to is finally called for.  That are my thoughts.  I was curious why you added the file.  Separate PR is fine with me.